src/css/base/_variables_coa.scss

```diff
@@ -1,8 +1,37 @@
+//typography
+@import url('https://fonts.googleapis.com/css?family=Lora|Open+Sans:400,600,700,800');
+/* TODO: determine fallback fonts */
+$coa-font-sans: 'Open Sans';
+$coa-font-serif: 'Lora';
+$coa-font-regular: 400;
+$coa-font-semibold: 600;
+$coa-font-bold: 700;
+$coa-font-extrabold: 800;
+$coa-em-base: 10px;
+$coa-base-font-size: 2rem;
+$coa-base-line-height: 2.5rem;
+
+$coa-font-size-xsmall: 1.4rem;
```

Comment:
did these come from the Sketch "sticker-sheet" page?

Reply:
it came from the sketchfile -- not necessarily the stickersheet. We can adjust as needed.
I was looking at the prototyping object that showed up on the right. 
I made some executive decisions on line-heights that were really close.

![screen shot 2018-03-19 at 6 48 41 pm](https://user-images.githubusercontent.com/3039125/37628175-5316ec5c-2ba6-11e8-8e42-e2d609a9bff4.png)